features/SPS007_Spatial-containment.feature

```diff
@@ -0,0 +1,23 @@
+@implementer-agreement
+@SPS
+@version1
+@E00040
+
+Feature: SPS007 - Spatial Containment
+The rule verifies that correct spatial containment - IfcRelContainedInSpatialStructure
+
+    Scenario: Subtypes of IfcElement are placed within the project spatial hierarchy
+        Given an IfcSpatialElement
+        Given A relationship IfcRelContainedInSpatialStructure from IfcSpatialElement to IfcElement and following that
+        Given is_a != IfcElementAssembly
+        Given ContainedInStructure = not empty
+        Given its attribute ContainedInStructure
+        Given its relating entity
+
+        Then The current directional relationship must not contain multiple entities at depth 1
+
```

Comment:
I don't understand how this relates to the concept template.  It appears to be validating the [1:1] cardinality of `IfcRelContainedInSpatialStructure.RelatingStructure` which would be flagged in the VS schema check that runs before the normative rules, no?